metricbeat/docs/fields.asciidoc

```diff
@@ -30514,10 +30500,10 @@ type: object
 
 --
 
-*`sql.query.metrics.string.*`*::
+*`sql.metrics.string.*`*::
 +
 --
-Non-numeric values collected.
+Non-numeric values collected. - release: beta
```

Comment:
something smells bad here

Reply:
Yep, I have needed a while to find the proper combination of spaces in the yamls, it should be fixed now.

Prometheus and statsd modules have the same problem, I will open a different PR to fix them.

Reply:
PR for the same change in other modules https://github.com/elastic/beats/pull/15742